tests/cfgs/default/result/sites.pcapng.out

```diff
@@ -111,8 +110,8 @@ JA3 Host Stats:
+	25	TCP 192.168.1.128:39302 <-> 95.131.170.91:443 [proto: 91/TLS][IP: 0/Unknown][Encrypted][Confidence: DPI][DPI packets: 6][cat: Web/5][3 pkts/849 bytes <-> 5 pkts/3703 bytes][Goodput ratio: 76/91][0.14 sec][Hostname/SNI: static.tuenti.com][(Advertised) ALPNs: h2;http/1.1][TLS Supported Versions: TLSv1.3;TLSv1.2][bytes ratio: -0.627 (Download)][IAT c2s/s2c min/avg/max/stddev: 43/0 46/24 50/49 4/24][Pkt Len c2s/s2c min/avg/max/stddev: 74/74 283/741 583/1514 218/647][TLSv1.2][JA3C: 579ccef312d18482fc42e2b822ca2430][JA4: t13d1715h2_5b57614c22b0_3d5424432f57][ServerNames: *.tuenti.com,tuenti.com][JA3S: 61be9ce3d068c08ff99a857f62352f9d][Issuer: C=US, O=DigiCert Inc, CN=DigiCert TLS RSA SHA256 2020 CA1][Subject: C=ES, L=Madrid, O=Tuenti Technologies S.L., CN=*.tuenti.com][Certificate SHA-1: 89:B8:FA:C7:22:04:D2:BE:C5:6E:59:10:31:67:42:B1:3F:6D:F8:3B][Firefox][Validity: 2021-04-05 00:00:00 - 2022-05-06 23:59:59][Cipher: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256][Plen Bins: 0,16,0,16,0,0,0,0,0,0,0,0,0,16,0,0,16,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,34,0,0]
+	26	TCP 192.168.1.128:51248 <-> 95.131.169.91:443 [proto: 91/TLS][IP: 0/Unknown][Encrypted][Confidence: DPI][DPI packets: 6][cat: Web/5][3 pkts/849 bytes <-> 5 pkts/3703 bytes][Goodput ratio: 76/91][0.14 sec][Hostname/SNI: tuenti.com][(Advertised) ALPNs: h2;http/1.1][TLS Supported Versions: TLSv1.3;TLSv1.2][bytes ratio: -0.627 (Download)][IAT c2s/s2c min/avg/max/stddev: 46/0 47/24 48/47 1/24][Pkt Len c2s/s2c min/avg/max/stddev: 74/74 283/741 583/1514 218/647][TLSv1.2][JA3C: 579ccef312d18482fc42e2b822ca2430][JA4: t13d1715h2_5b57614c22b0_3d5424432f57][ServerNames: *.tuenti.com,tuenti.com][JA3S: 61be9ce3d068c08ff99a857f62352f9d][Issuer: C=US, O=DigiCert Inc, CN=DigiCert TLS RSA SHA256 2020 CA1][Subject: C=ES, L=Madrid, O=Tuenti Technologies S.L., CN=*.tuenti.com][Certificate SHA-1: 89:B8:FA:C7:22:04:D2:BE:C5:6E:59:10:31:67:42:B1:3F:6D:F8:3B][Firefox][Validity: 2021-04-05 00:00:00 - 2022-05-06 23:59:59][Cipher: TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256][Plen Bins: 0,16,0,16,0,0,0,0,0,0,0,0,0,16,0,0,16,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,34,0,0]
```

Comment:
Could you try to remove these 2 flows from the trace, please?

Reply:
> Could you try to remove these 2 flows from the trace, please?

Yea, sure.